### ISSUE TITLE
Circuit Browser Improvement

### ISSUE THREAD

**Issue by blane1257** [CircuitVerse/CircuitVerse#30]
There should be a "Most Viewed" section to the home page, or maybe a proper circuit browser with a few options for filtering and order.

**Comment by vedant-jain03** [comment]
@tachyons I want to work on this issue. May I?

**Comment by SannidhiNair** [comment]
@tachyons is this issue still valid?

**Comment by tachyons** [comment]
Yes, but we need to merge with similar issues and make detailed one

**Comment by SannidhiNair** [comment]
Okay so I will reference similar issues and combine them in one and make a new one, and close others.

**Comment by ThatDeparted2061** [comment]
@SannidhiNair 
@tachyons 
Most viewed part is a feature request, a decent one , I think this would come under Project 3 in the gsoc ideas so we can remove it for the time being

The filtering part is the same as the other issues asking about improvemnt on searching algorithm, so we can shift this there too

**Comment by salmoneatenbybear** [comment]
This will be under the scope of the circuit explore page. 

Cc : @VaibhavUpreti @yashikajotwani12
